python/py-nibabel/Portfile

```diff
@@ -4,36 +4,51 @@ PortSystem              1.0
 PortGroup               python 1.0
 
 name                    py-nibabel
-version                 2.3.3
+version                 2.4.0
 revision                0
 platforms               darwin
 license                 MIT
 maintainers             nomaintainer
 description             Access a cacophony of neuro-imaging file formats
 
-long_description        Nibabel provides read and write access to some common medical and \
-    neuroimaging file formats, including: ANALYZE (plain, SPM99, SPM2), GIFTI, NIfTI1, MINC, \
-    MGH and ECAT as well as PAR/REC. We can read and write Freesurfer geometry, and read \
-    Freesurfer morphometry and annotation files. There is some very limited support for \
-    DICOM. NiBabel is the successor of PyNIfTI.
+long_description        Nibabel provides read and write access to some \
+                        common medical and neuroimaging file formats, \
+                        including: ANALYZE (plain, SPM99, SPM2), GIFTI, \
+                        NIfTI1, MINC, MGH and ECAT as well as PAR/REC. We \
+                        can read and write Freesurfer geometry, and read \
+                        Freesurfer morphometry and annotation files. There \
+                        is some very limited support for DICOM. NiBabel is \
+                        the successor of PyNIfTI.
 
 homepage                https://nipy.org/nibabel
 master_sites            pypi:n/nibabel/
 distname                nibabel-${version}
 
-checksums               rmd160  1f202d10b89872cb056f64334f262cfa02df0075 \
-                        sha256  b6366634c65b04464e62f3a9a8df1faa172f780ed7f1af1c6818b3dc2f1202c3 \
-                        size    4492243
+checksums \
+    rmd160  a479afa81c5b79752ef0f39dcde0304e53a26f3a \
```

Comment:
I would consider keeping your whitespaces consistent throughout the portfile

Reply:
I like to keep it under 80 columns. Personal preference. :)